liota/dccs/iotcc.py

```diff
@@ -89,6 +90,15 @@ def register(self, entity_obj):
         """ Register the objects
 
         """
+        if not self._validate_input(entity_obj.name):
```

Comment:
I think we can now modify the function _validate_input(input, length), check the matching string, check the size and throw a common ValueError, size mismatch or invalid character used in input "". The caller can just make a single call like _validate_input(entity_obj.name, 50).

Currently, the code seems to have a lot of duplication, just to print exactly where the issue is happening.

Reply:
@pmasrani  I have addressed the suggested changes. Please review the updated PR.